src/csi-wrapper/pkg/wrapper/nodeservice.go

```diff
@@ -11,7 +11,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/proto/podvminfo"
+	podvminfo "github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/proto/podvminfo"
```

Comment:
Is this necessary by this commit or deserves its own?

Reply:
I've updated the commit message to explain this. If we can get the package generated then that would be even better, but I know I had great fun getting this far at the end of last year.